packages/ui-kit/src/components/PieChart/PieChart.test.tsx

```diff
@@ -0,0 +1,173 @@
+import { render, waitFor } from '@testing-library/react'
+import { Chart } from 'chart.js'
+import React from 'react'
+import { RelativeTimeRange } from '../../helpers'
+import { Dom, mockLeaderboardQuery, mockCounterQuery, setupTestHandlers } from '../../testing'
+import { PieChart } from './PieChart'
+
+const mockCounterData = {
+  value: '2375'
+}
+
+const mockData = {
+  headers: ['DATA_SOURCE_TYPE', 'value'],
+  rows: [
+    ['Http', '250'],
+    ['Snowflake', '385'],
+    ['S3', '445'],
+    ['Redshift', '560']
+  ]
+}
```

Comment:
Can you use a different example? For example, "Taco Type"? And have "Carnitas", "Al Pastor", "Carne Asada", etc? Let's not reference Data Source types in any examples. Gets too "meta".